.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     },
   },
   rules: {
+    'no-case-declarations': 'off',
```

Comment:
This rule is not helpful, it encourages to obfuscate code, it enforces sth like this:

``` diff
+ const temp = foo + 5;
 switch (foo) {
     case 1:
-        const temp = foo + 5;
        return temp;
}
```

Reply:
is this change related to this PR?